src/platform/user/authentication/utilities.js

```diff
@@ -75,12 +68,25 @@ function redirectWithGAClientId(redirectUrl) {
   }
 }
 
+export function standaloneRedirect() {
+  const searchParams = new URLSearchParams(window.location.search);
+  const application = searchParams.get('application');
+  const to = searchParams.get('to');
+  let url = externalRedirects[application] || null;
+
+  if (url && to) {
+    url = url.replace(new RegExp('/$'), '');
```

Comment:
If we're going to cut off the trailing slash, do we need it in the first place in `externalRedirects`?

Reply:
no we can remove it, the thought was to leave it here just in case someone later adds an externalRedirect with a trailing slash.